sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -1332,14 +1325,14 @@
           {
             "name": "jobType",
             "in": "query",
-            "description": "Job Type.",
+            "description": "The job type.",
```

Comment:
Is this an enum? Do we know the supported values? We should ask the service team.

Reply:
I believe two possible values:

scheduleUpdateTwin: A job used to update a set of desired properties or tags.
scheduleDeviceMethod: A job used to invoke a device method on a set of device twins.

This is from this doc: https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-jobs#jobs-to-update-device-twin-properties

Should these possible values usually go in the description?

Reply:
If we have pre-defined values, we should mention/ link it in our description.

Reply:
Link is good, but would not be good to add it in description. It will get out of date when new things are added.